### ISSUE TITLE
Invoker dependency is undefined

### ISSUE THREAD

**Issue by ceilfors** [ceilfors/laconia#11]
When combining @laconia/invoker CoC and @laconia/core IoC destructuring, it is difficult to know why my dependency is undefined:

Environment variable:
```
LACONIA_INVOKER_CAPTURE_CARDPAYMENT: captureCardPayment
```

Code
```
module.exports.handler = laconiaBatch(
  _ =>
    laconiaBatch.dynamoDb({
...
    })
)
  .register(invoker.envVarInstances())
  .on("item", ({ captureCardPayment }, item) =>
    captureCardPayment.fireAndForget({
      paymentReference: item.paymentReference
    })
  );
```

Error message:
```
2018-12-13T06:56:13.417Z	2d4b47ea-fea4-11e8-8be0-23ecc03cd47f
{
    "errorMessage": "Cannot read property 'fireAndForget' of undefined",
    "errorType": "TypeError",
    "stackTrace": [
        "AsyncFunction.module.exports.handler.laconiaBatch.register.postProcessor.on (/var/task/src/process-card-payments.js:18:23)",
        "emitTwo (events.js:126:13)",
        "AsyncFunction.emit (events.js:214:7)",
        "BatchProcessor.from.on.args (/var/task/node_modules/@laconia/batch/src/laconiaBatch.js:8:40)",
        "emitOne (events.js:116:13)",
        "BatchProcessor.emit (events.js:211:7)",
        "BatchProcessor.start (/var/task/node_modules/@laconia/batch/src/BatchProcessor.js:27:14)",
        "<anonymous>"
    ]
}

```

Various causes:
- There is a missing underscore in the environment variable name LACONIA_INVOKER_CAPTURE_CARDPAYMENT.
- I might forget to register invoker.envVarInstances()

**Comment by ceilfors** [comment]
@andrew-modelsoftware Does this capture the issue you reported correctly?

**Comment by ceilfors** [comment]
The error will be more explicit like this now:

```
2018-12-15T21:02:29.528Z	bba331c0-00ac-11e9-82f5-1995c871f6ab
{
    "errorMessage": "The dependency captureCardPayment is not available. Have you registered your dependency? These are the dependencies available in LaconiaContext: ..., captureCardpayment",
    "errorType": "Error",
    "stackTrace": [
        "checkInstanceName (/var/task/node_modules/@laconia/batch/node_modules/@laconia/core/src/LaconiaContext.js:21:11)",
        "Object.get (/var/task/node_modules/@laconia/batch/node_modules/@laconia/core/src/LaconiaContext.js:34:7)",
        "AsyncFunction.module.exports.handler.laconiaBatch.register.postProcessor.on (/var/task/src/process-card-payments.js:17:18)",
        "emitTwo (events.js:126:13)",
        "AsyncFunction.emit (events.js:214:7)",
        "BatchProcessor.from.on.args (/var/task/node_modules/@laconia/batch/src/laconiaBatch.js:8:40)",
        "emitOne (events.js:116:13)",
        "BatchProcessor.emit (events.js:211:7)",
        "BatchProcessor.start (/var/task/node_modules/@laconia/batch/src/BatchProcessor.js:27:14)",
        "<anonymous>"
    ]
}
```

**Comment by ceilfors** [comment]
Released under 0.15.0
